Tables to summarize various integral schemes in pbc scf (#42)
pyscf/pyscf.github.io: source/user/pbc/scf.rst

    kmf = scf.KRHF(cell).run()
    kmf.stability()

.. _user_pbc_multigrid:
Multigrid
---------
For pure DFT calculations, multi-grid algorithm becomes efficient if the size of
unit cell is relatively large (e.g. more than 10 atoms) or the kinetic energy

> cutoff is relatively high (e.g. more than 100k plan-ewaves). This algorithm is

Reviewer: ```suggestion
cutoff is relatively high (e.g. more than 100k plane waves). This algorithm is
```

implemented in the :mod:`pyscf.pbc.dft.multigrid` module::

  from pyscf.pbc.dft import multigrid
  mf = multigrid.multigrid(dft.KRKS(cell))
  mf.run()

More examples can be found

.. literalinclude:: ../../examples/pbc/27-multigrid.py

> Multi-grid algorithm does not support exact exchange.

Reviewer: ```suggestion
The multi-grid algorithm does not support exact exchange.
```

.. _user_pbc_rsjk:
Range-separation integration
----------------------------

> This algorithm computes most of four-center integrals in real space. For small

Reviewer: ```suggestion
 This algorithm computes most of the four-center integrals in real space. For small
```
